ParisBNF/abb/BNFabb64.xml

```diff
@@ -197,11 +197,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                                          <item xml:id="a5">
                                            <locus target="#172v"/>
-                                           <desc type="PoemQene"><term key="Qene">qǝne-poem</term> of the type <term key="Mawaddes">mawaddǝs</term>,
-                                           of mamhǝr Zawalda Giyorgis, written in the same crude hand as the previous notes,
-                                         poorly readable</desc>
+                                           
+                                           <desc type="PoemQene"><title type="complete" ref="LIT7006Yezarreu">A <term key="Qene">qǝne-poem</term> of <term key="Qene">mawaddǝs type</term>, 
```

Comment:
better close the title `<title type="complete" ref="LIT7006Yezarreu"/> ` , hyperlink inside of a hyperlink is not possible

Reply:
I did (before this structure used to work).